cmd/minikube/cmd/version.go

```diff
@@ -74,4 +123,5 @@ var versionCmd = &cobra.Command{
 func init() {
 	versionCmd.Flags().StringVarP(&versionOutput, "output", "o", "", "One of 'yaml' or 'json'.")
 	versionCmd.Flags().BoolVar(&shortVersion, "short", false, "Print just the version number.")
+	versionCmd.Flags().BoolVar(&listPackagesVersions, "packages", false, "list versions of all packages included with minikube. (cluster must be running)")
```

Comment:
`all packages included with minikube` is not precise here, we have more minikube specific packages like cni and podman, and also a lot of packages inherited from distro

Reply:
Calling this "packages" is currently a bit misleading, since we don't output the package version (deb) but the software version

Reply:
sounds good, I am open to change, any suggestion for better wording ?